modules/amazon/index.js

```diff
@@ -0,0 +1,61 @@
+var amazon = require('amazon-product-api');
+var amazonClient = null;
+var log = bot.log;
```

Comment:
bot isn't defined here, so this needs to be split across `var log` here, and `log = ` in init.